harness/determined/cli/trial.py

```diff
@@ -494,6 +504,22 @@ def create_json_file_in_dir(content: Any, file_path: str) -> None:
                     *logs_args_description,
                 ],
             ),
+            Cmd(
+                "retain-logs",
```

Comment:
Same point as in `experiment.py`.
`days` should be a positional argument that is `int_or_none` and then `--forever` can be an optional argument.

Reply:
done, also added checks to ensure that only one argument is provided